source/usage-examples/insertOne.txt

```diff
@@ -0,0 +1,27 @@
+=================
+Create a Document
+=================
+
+.. default-domain:: mongodb
+
+You can create a single document using the `insertOne()
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#insertOne>`_
+method. The ``insertOne()`` method inserts a document into the specified collection
+using a document object. If the specified collection does not exist, the ``insertOne()``
+method creates the collection. 
+
+Create an `Object
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+to specify additional options. Set the ``bypassDocumentValidation`` option to bypass
+:manual:`schema validation <core/schema-validation/>` rules in MongoDB 3.2 or higher.
```

Comment:
- This should use "You can..." to maintain the same mood as the rest of the example (potential mood vs imperative mood)
- Is `bypassDocumentValidation` a core option? Not sure which options we want to include/exclude, but if we're only including essential options this seems extraneous

Reply:
I've removed this paragraph altogether since there are no typical/essential options that we would want for a usage-example of insertOne.

I've also removed `bypassDocumentValidation` from insertMany per Nathan's suggestion offline that we should not tell our users to bypass schema validation